docs/src/main/sphinx/release/release-446.md

```diff
@@ -0,0 +1,65 @@
+# Release 446 (29 Apr 2024)
+
+## General
+
+* Improve performance of `INSERT` statements into partitioned tables when the
+  `retry_policy` configuration property is set to `TASK`. ({issue}`21661 `)
+* Reduce delay before killing queries when the cluster runs out of memory. ({issue}`21719`)
+* Fix `CREATE CATALOG` statements including quotes in catalog names. ({issue}`21399`)
+* Fix potential query failure when a column name ends with a `:`. ({issue}`21676`)
+* Fix potential query failure when a [SQL routine](/routines) contains a label
+  reference in a `LEAVE`, `ITERATE`, `REPEAT`, or `WHILE` statement. ({issue}`21682`)
+* Fix query failure when [SQL routines](/routines) use special functions such as
+  `NULLIF` or `BETWEEN`. ({issue}`19820`)
+* Fix potential query failure due to worker nodes running out of memory in
+  concurrent scenarios. ({issue}`21706`)
+
+## BigQuery connector
+
+* {{breaking}} Enable `bigquery.arrow-serialization.enabled` by default.
+  `--add-opens=java.base/java.nio=ALL-UNNAMED` must be added to`jvm-config`. ({issue}`21580`)
+
+## Delta Lake connector
+
+* Fix failure when reading from Azure file storage and the schema, table, or
+  column name contains non-alphanumeric characters. ({issue}`21586`)
+* Fix incorrect results when reading a partitioned table with a
+  [deletion vector](https://docs.databricks.com/en/delta/deletion-vectors.html). ({issue}`21737`)
+
+## Hive connector
+
+* Add support for reading S3 objects restored from Glacier storage. ({issue}`21164`)
+* Fix failure when reading from Azure file storage and the schema, table, or
+  column name contains non-alphanumeric characters. ({issue}`21586`)
+
+## Iceberg connector
+
+* Add support for the [Snowflake catalog](iceberg-snowflake-catalog). ({issue}`19362`)
+* Automatically use `varchar` as a type during table creation when `char` is
+  specified. ({issue}`19336`, ({issue}`21515`)
+* Deprecate the `schema` and `table` arguments for the `table_changes` function
+  in favor of `schema_name` and `table_name`, respectively. ({issue}`21698`)
+* Prevent assigning null values to non-null columns as part of a `MERGE`
+  statement. ({issue}`21619`)
```

Comment:
The PR fixed the engine, not Iceberg connector. I think all connectors supporting MERGE statement & NOT NULL constraints are affected?